### PR TITLE
[VEN-2312] PSR Configuration on opbnbmainnet

### PR DESCRIPTION
**Pull request by 0xlucian**
## Description

This PR adds VIP and simulation for configuring `PSR` on `opbnbmainnet`

Resolves VEN-2312

### CODE REVIEW

**Review by chechu — APPROVED** [review]

File: multisig/proposals/vip-005/vip-005-opbnbmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  return makeProposal([
+    {
+      target: PROTOCOL_SHARE_RESERVE,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    {
+      target: PROTOCOL_SHARE_RESERVE,
+      signature: "setAccessControlManager(address)",
+      params: [ACM],
```
Comment (GitGuru7) [inline multisig/proposals/vip-005/vip-005-opbnbmainnet.ts:50]:
```suggestion

```
ACM is already configured

File: multisig/simulations/vip-005/vip-005-opbnbmainnet/simulations.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+const { opbnbmainnet } = NETWORK_ADDRESSES;
+
+export const ACM = "0xA60Deae5344F1152426cA440fb6552eA0e3005D6";
+export const PROTOCOL_SHARE_RESERVE = "0xA2EDD515B75aBD009161B15909C19959484B0C1e";
+export const POOL_REGISTRY = "0x345a030Ad22e2317ac52811AC41C1A63cfa13aEe";
+export const VTREASURY = "0xDDc9017F3073aa53a4A8535163b0bf7311F72C52";
+export const OPBNBMAINNET_MULTISIG = opbnbmainnet.NORMAL_TIMELOCK;
+
+export const MARKETS: VenusMarket[] = [
```
Comment (GitGuru7) [inline multisig/simulations/vip-005/vip-005-opbnbmainnet/simulations.ts:22]: Can be import from VIP file or we can remove export.

**Review by GitGuru7 — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Review by kkirka — APPROVED** [review]
LGTM: opbnbmainnet addresses are correct, distribution config and access controls look good.

**Comment by chechu** [comment]
https://opbnbscan.com/tx/0x3c8505eca9ffa18e564d9f837f8f27b701323850c3679e8d165a3849b666dc8d

**Pull request merged**